flow/IndexedSet.h

```diff
@@ -713,7 +713,7 @@ int IndexedSet<T,Metric>::insert(const std::vector<std::pair<T,Metric>>& dataVec
 			bool foundNode = false;
 			while (true) {
 				int cmp = compare(data, t->data);
-				d = cmp >= 0;
+				d = cmp > 0;
 				if (!d)
```

Comment:
I think this would read more clearly as `if (d ==0)` since d represents an index into `child`.

Reply:
agreed.  fixing + then I'll merge it.